README.md

```diff
@@ -48,6 +48,11 @@ To run `znap pull` on specific repos only, including ones you have set to be exc
 % znap pull <repo> ...
 ```
 
+To be prompted to automatically update with `znap pull` every 30 days, add the following to your `.zshrc` file:
+```sh
+zstyle ':znap:pull' auto-update 30
```

Comment:
I'm more than happy to take feedback on naming of the zstyle -- not sure what the most consistent with the other styles would be. This is different in that it doesn't have a repo name as part of it. Also not sure if this should include "interval" or "days" or something as part of the name.

Also, it could be possible to have styles on whether or not to prompt.